tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -153,11 +153,11 @@ def test_a_flag(tmp_path, manim_cfg_file, infallible_scenes_path):
         sys.executable,
         "-m",
         "manim",
-        infallible_scenes_path,
         "-ql",
         "--media_dir",
         str(tmp_path),
         "-a",
+        infallible_scenes_path,
```

Comment:
Why did the path to the scenes file need to be moved to the end of the command?

Reply:
Because 

> ## Overview / Explanation for Changes
> 
> This moves Manim's CLI handling to be more in line with POSIX compliant conventions. It also means that options for commands are given BEFORE their arguments which may be a source of confusion for some, but it very easy to adapt to... instead of `manim basic.py -pql`, the command becomes `manim -pql basic.py`

Reply:
> edit: This question is meant more to improve my own knowledge than to be considered as a "review" like GitHub labeled it.

No worries! Feel free to actually review the code if you'd like (my code isn't in cleanest condition so it's preferred really), but also feel free to make general comments/ask questions/ping me on Discord about stuff. I'm still learning more about Click to this day so it may even improve my understanding of the library's capabilities.